media/js/base/mozilla-monitor-button.js

```diff
@@ -20,17 +20,41 @@ Mozilla.MonitorButton = (function() {
     }
 
     var buttonURL = monitorButton.getAttribute('href');
+    // strip url to everything after `?`
+    buttonURL = buttonURL.match(/\?(.*)/)[1];
```

Comment:
Suggest storing the regex match in a variable here as opposed to setting `buttonURL`. Further down on line 55 we have:

```javascript
monitorButton.setAttribute('href', buttonURL);
```

So currently the monitor button is being turned into a relative link, vs sending people to monitor.firefox.com.